test/com/amazon/ionschema/SchemaImportTest.kt

```diff
@@ -46,6 +46,14 @@ class SchemaImportTest {
         }
     }
 
+    @Test
+    fun getImport_new_schema_from_symbol() {
```

Comment:
To make it slightly more obvious what's being tested, how about:

```suggestion
    fun getImport_id_is_a_symbol() {
```